packages/core/core/src/PackagerRunner.js

```diff
@@ -613,29 +613,43 @@ export default class PackagerRunner {
     cacheKeys: CacheKeyMap,
     type: string,
     contents: Blob,
-    map: ?Blob,
+    map: ?string,
   ): Promise<BundleInfo> {
     let size = 0;
     let hash = crypto.createHash('md5');
-    let boundaryStr = '';
     let hashReferences = [];
-    await this.options.cache.setStream(
-      cacheKeys.content,
-      blobToStream(contents).pipe(
-        new TapStream(buf => {
-          let str = boundaryStr + buf.toString();
-          hashReferences = hashReferences.concat(
-            str.match(HASH_REF_REGEX) ?? [],
-          );
-          size += buf.length;
-          hash.update(buf);
-          boundaryStr = str.slice(str.length - BOUNDARY_LENGTH);
-        }),
-      ),
-    );
+
+    // TODO: don't replace hash references in binary files??
```

Comment:
I guess this wasn't introduced by your changes?

Reply:
Yeah just a problem I noticed. But not sure how to know which files we should do this for.

Reply:
We use this in the data url optimizer, but its detection logic is rather sketchy : https://github.com/gjtorikian/isBinaryFile/issues/22